src/main/java/model/Head.java

```diff
@@ -1,4 +1,70 @@
 package model;
 
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.StringReader;
+import java.util.HashMap;
+import java.util.Map;
+
 public class Head {
```

Comment:
```suggestion
public class HttpRequestHead {
```
헤드를 객체로 만든 건 좋은 시도 같습니다.